zokrates_js/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM blockchainit/rust-wasm-env:latest
```

Comment:
What are we using Docker for for zokrates_js?

Reply:
It's not needed anymore